rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json

```diff
@@ -1,20 +1,29 @@
 [
-    {
-        "domain": 1,
-        "gasOracle": {
-            "type": "remoteGasData",
-            "tokenExchangeRate": "15000000000000000000",
-            "gasPrice": "10000000000",
-            "tokenDecimals": 18
-        }
-    },
-    {
-        "domain": 1399811149,
-        "gasOracle": {
-            "type": "remoteGasData",
-            "tokenExchangeRate": "887000000000000000",
-            "gasPrice": "15",
-            "tokenDecimals": 9
-        }
+  {
+    "domain": 1,
+    "gasOracle": {
+      "type": "remoteGasData",
+      "tokenExchangeRate": "15000000000000000000",
+      "gasPrice": "10000000000",
+      "tokenDecimals": 18
     }
+  },
+  {
+    "domain": 1399811149,
+    "gasOracle": {
+      "type": "remoteGasData",
+      "tokenExchangeRate": "887000000000000000",
+      "gasPrice": "15",
+      "tokenDecimals": 9
+    }
+  },
+  {
+    "domain": 745,
+    "gasOracle": {
+      "type": "remoteGasData",
+      "tokenExchangeRate": "46951091051",
+      "gasPrice": "0.025",
```

Comment:
I think gasPrice must be an integer so we need to do some precision adjustments

Using a test I came up with these numbers, wdyt?

```
    #[test]
    fn test_quote_gas_payment() {
        let mut igp = Igp::default();
        let stride_domain = 745;

        igp.gas_oracles.insert(
            stride_domain,
            GasOracle::RemoteGasData(RemoteGasData {
                // ((0.7331 / 2526.50) * 1e19 (exchange rate scale) * 1.5 (apply premium)) / 10 (scaling down to let the gas price have more precision)
                token_exchange_rate: 435246388284187,
                // Scaling up the gas price to reach a desired cost
                gas_price: 7,
                token_decimals: 6,
            }),
        );

        let quote = igp.quote_gas_payment(stride_domain, 300000).unwrap();
        println!("quote: {}", quote);
  }
```

which gives a quote of `91000` eth lamports, which is `(91000 / 1e9) * 2526 = 23 cents`